src/pages/main/components/carousel/Carousel.js

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from 'react';
+import CarouselImg from './CarouselImg';
+import './Carousel.scss';
+
+const Carousel = () => {
+  const [imgList, setImgList] = useState([]);
+  const [curImg, setCurImg] = useState(0);
+  const [fadeIn, setFadeIn] = useState(true);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/CAROUSEL_DATA.json', {
```

Comment:
```suggestion
    fetch('/data/CAROUSEL_DATA.json', {
```
http://localhost:3000 은 생략 가능합니다. 포트 번호가 달라지면(ex. localhost:3001) 코드 전체를 바꿔야 하기 때문에 수정해주시기 바랍니다.

Reply:
수정했습니다!